antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+set -e
+
+BASEDIR=$(dirname "$(readlink -f "$0")")
+
+CURRENT_TAG=${git describe --tag}
+
+createReleaseDocumentation() {
+  "${BASEDIR}"/execute_github_site_maven_plugin.sh
+}
+
+deleteOldSnapshotFolderInGhPages() {
+  fetchGhPages
+  git checkout gh-pages
+  OLD_SNAPSHOT_DIR=$(ls -d *SNAPSHOT*/)
+  rm -rf "*SNAPSHOT*"
+  git commit -am "Remove ${OLD_SNAPSHOT_DIR} documentation folder"
+  pushGhPages
+  git checkout ${CURRENT_TAG}
+}
+
+createCurrentSnapshotDocumentation() {
+  git checkout master
+  "${BASEDIR}"/execute_github_site_maven_plugin.sh
+  git checkout ${CURRENT_TAG}
+}
+
+createNewIndexRootFile() {
+  fetchGhPages
+  "${BASEDIR}"/.writeIndexRootFile.sh
+  pushGhPages
+}
+
+fetchGhPages() {
+  git fetch "https://${GITHUB_CREDENTIALS_USR}:${GITHUB_CREDENTIALS_PSW}@github.com/eclipse/antenna.git" gh-pages:gh-pages
```

Comment:
fetch should work without credentials:
```suggestion
  git fetch "https://github.com/eclipse/antenna.git" gh-pages:gh-pages
```